packages/plugin-connector/src/data-sources/exec-source.js

```diff
@@ -46,7 +46,6 @@ export const execSource = async (source, supportedDbs, outDir) => {
 		const parquetBuffer = await buildParquetFromResultSet(result.columnTypes, result.rows);
 		/* Split on / or \ (windows compatibility) */
 		const fileparts = query.filepath.split(/[/\\]/);
-		const outputFilename = fileparts.pop()?.split('.')[0];
```

Comment:
Good catch, made me notice something else in the `outputSubdir` logic:
```js
const outputSubdir = path.join(
	...path
		.join(...fileparts)
		.split('sources')
		.slice(1)
);
```

If `sources` is somewhere else in the file path, like `/Users/john_mcsources/my_project/sources/needful_things/orders.sql` the file path would end up as `/my_project/needful_things` instead of the intended `/needful_things`. Not a major issue, since data file path doesn't *really* matter afaict, but worth nothing